proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -896,9 +820,9 @@ message IndividualVotesRespond {
         // The current effective balance of the validator.
         uint64 current_epoch_effective_balance_gwei = 13;
         // The slots of when the validator's attestation got included in the block. Only available in phase0.
-        uint64 inclusion_slot = 14 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v4/consensus-types/primitives.Slot", deprecated = true];
```

Comment:
wait why can't we remove this?

Reply:
and the thing below

Reply:
because these fields are used in `GetIndividualVotes` and there doesn't seem to be a replacement for them (that being said, the whole `GetIndividualVotes` is not used anywhere and should be marked as deprecated).